api/desecapi/pdns.py

```diff
@@ -18,6 +18,9 @@
 
     # "additional" types, without obsolete ones
     'DHCID', 'DLV', 'EUI48', 'EUI64', 'IPSECKEY', 'KX', 'MINFO', 'MR', 'RKEY', 'WKS',
+
+    # https://doc.powerdns.com/authoritative/changelog/4.5.html#change-4.5.0-alpha1-New-Features
+    'NID', 'L32', 'L64', 'LP'
```

Comment:
aren't those 'major' types? (This classification originally stems from the pdns docs.)

Reply:
No, [they aren't listed in the pdns docs](https://doc.powerdns.com/authoritative/appendices/types.html), but they are announced in the 4.5 release notes. So I made a new category here.

I just submitted [a PR for documentation](https://github.com/PowerDNS/pdns/pull/10607).